clients/src/main/java/org/apache/kafka/clients/telemetry/DefaultClientTelemetry.java

```diff
@@ -336,7 +337,7 @@ public void telemetrySubscriptionSucceeded(GetTelemetrySubscriptionsResponseData
         List<CompressionType> acceptedCompressionTypes = validateAcceptedCompressionTypes(data.acceptedCompressionTypes());
         Uuid clientInstanceId = validateClientInstanceId(data.clientInstanceId());
         // TODO: TELEMETRY_TODO: this is temporary until we get real data back from broker...
```

Comment:
Can you move this comment into `getPushIntervalMs` so that we don't forget to take out the hardcoded `10000` ms. value?